src/main/java/edu/harvard/iq/dataverse/DataFile.java

```diff
@@ -57,6 +57,10 @@
 @NamedQueries({
 	@NamedQuery( name="DataFile.removeFromDatasetVersion",
 		query="DELETE FROM FileMetadata f WHERE f.datasetVersion.id=:versionId and f.dataFile.id=:fileId"),
+        @NamedQuery(name = "DataFile.findByCreatorId",
```

Comment:
I noticed these are repeated for all the dvobjects? Could we simplify by just getting from dvobject? (is there a case for individually? it just seems a little over kill)

Reply:
ok.